### PR TITLE
DSD-1494 & DSD-1495: "connect" section colors and Breadcrumbs variant

### PR DESCRIPTION
**Pull request by bigfishdesign13**
Fixes JIRA ticket [DSD-1494](https://jira.nypl.org/browse/DSD-1494) & [DSD-1495](https://jira.nypl.org/browse/DSD-1495)

## This PR does the following:

- Adds the `"connect"` section hues to the light and dark color palettes.
- Adds the `"connect"` color variant to the `Breadcrumbs` component.

## How has this been tested?

- local build of Storybook

## Accessibility concerns or updates

- The new "connect" colors and color combinations were vetted for a11y compliance.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/theme/foundations/colors.ts [hunk 1]
```diff
@@ -40,6 +40,8 @@ const brandPrimary = "#C60917";
 const brandSecondary = "#760000";
 const blogsPrimary = grayLightCool;
 const blogsSecondary = grayMedium;
+const connectPrimary = "#737373";
```
Comment (EdwinGuzman) [inline src/theme/foundations/colors.ts:43]: Just curious, why are these two gray-ish colors not based on the existing gray color palette?

File: src/theme/foundations/colors.ts [hunk 2]
```diff
@@ -276,6 +290,10 @@ const colors: Colors = {
      * New Arrivals, and E-Book Central. */
     "books-and-more": booksAndMoreObj.light,
 
+    /** Books and More is used for the Catalog, Staff Picks, Recommendations,
```
Comment (EdwinGuzman) [inline src/theme/foundations/colors.ts]: The comment should be updated.

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
Just a minor update to update the comment.

File: src/theme/foundations/colors.ts (hunk 1 above)
Comment (bigfishdesign13) [reply src/theme/foundations/colors.ts:43]: In a way, they are branded colors.  They were selected by the creative director.

**Review by EdwinGuzman — APPROVED** [review]

**Pull request merged**